components/org.wso2.carbon.identity.api.server.userstore/org.wso2.carbon.identity.api.server.userstore.v1/src/main/java/org/wso2/carbon/identity/api/server/userstore/v1/core/ServerUserStoreService.java

```diff
@@ -1011,6 +1227,25 @@ private void validateUserstoreUpdateRequest(String domainID, UserStoreReq userSt
         }
     }
 
+    /**
+     * Validate userstore domain Id.
+     *
+     * @param userstoreDomainId    Userstore domain ID
```

Comment:
Let's add a `.` at the end

Reply:
Addressed with commit c731518fd0ea44d2d06c025d044e61114470